schema/badges.ts

```diff
@@ -0,0 +1,51 @@
+import { z } from 'zod';
+import { dateSchema } from '../util/sharedSchema';
+
+export const VERSION = '1.0.0';
+
+export const schema = z
+  .object({
+    badgeId: z.string(),
+    name: z.string(),
+    displayName: z.string(),
+    description: z.string(),
```

Comment:
nit: we can add comments `/** */` before the field definitions. The comments we write will appear in the inferred Typescript type (I have no idea how it worked -- must be magic)
![image](https://github.com/user-attachments/assets/82dc6859-df94-4662-a4e0-f8df5f9b27f9)

I think it is worth noting that
- `displayName` is the compact version of `name`
- `description` can contain Markdown syntax

Reply:
nit: thanks for adding the comments. Personally I prefer putting comments of individual fields on top of the field directly, so that it is easier to see.

Typescript JSDoc resolution also works better for the field when we put comment of individual field right before the field itself.